specs/wp-invite-users-spec.js

```diff
@@ -206,6 +206,7 @@ describe( `[${ host }] Invites:  (${ screenSize })`, function() {
 		const newInviteEmailAddress = dataHelper.getEmailAddress( newUserName, inviteInboxId );
 		const siteName = config.get( 'privateSiteForInvites' );
 		const siteUrl = `https://${ siteName }/`;
+		let removedViewerFlag = false;
```

Comment:
This flag is used in `after` block - if the user is successfully deleted in `Can see new user added and can be removed` step, there is no need to try deleting user again.